src/test/java/com/google/api/codegen/testsrc/common/library.proto

```diff
@@ -753,13 +753,17 @@ message MoveBookRequest {
 }
 
 message ListStringsRequest {
-  string name = 1;
+  string name = 1 [
```

Comment:
This is what the other library_gapic.yaml say to do: https://github.com/googleapis/gapic-generator/blob/5bad4a26f0b4ffe26c7e0f4b5559482ad6c8af3f/src/test/java/com/google/api/codegen/testsrc/common/library_gapic.yaml#L1187